test.rb

```diff
@@ -1,6 +1,6 @@
 class TestClass
 
-	def foo
-	puts 'bar'
-	end
+def foo
+puts 'bar'
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.